src/main/java/com/booster/server/NoteEntity.java

```diff
@@ -0,0 +1,40 @@
+package com.booster.server;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.Hibernate;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.Objects;
+
+@Table(name = "note")
+@Entity
+@Getter
+@Setter
+public class NoteEntity {
+
+    @Id
+    @GeneratedValue
+    @EqualsAndHashCode.Include
```

Comment:
No need for the `@EqualsAndHashCode.Include` since we have custom `equals` and `hashcode` methods.

Reply:
Done